packages/sdk/lib/base.ts

```diff
@@ -220,6 +220,13 @@ class Itly {
       return;
     }
 
+    if (userId != null && typeof (userId) === 'object') {
```

Comment:
For TS I think we need to change the method signature to allow `userId` to be a Properties object right?

Reply:
Also this should be moved to the browser.ts instead of base.ts. For node and base we still want to require the first param to be a userId string.

Reply:
Moved